template/package.json

```diff
@@ -27,7 +27,6 @@
     "@alaskaairux/design-tokens": "",
     "@alaskaairux/webcorestylesheets": "",
     "@webcomponents/webcomponentsjs": "",
-    "focus-visible": ""
```

Comment:
Please remove the trailing `,` on line 29. Leaving this caused the build to fail. Also, please `amend` the previous commit. 

1. make update
2. `git add --all`
3. `git commit --amend`

Reply:
Also, please remove this line 

- https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/src/%5Bnamespace%5D-%5Bname%5D.js#L15

Reply:
will do, thanks @blackfalcon.